docs/reference/config.md

```diff
@@ -258,7 +258,7 @@ git:
   # status of the current repository. If a change is detected that is 
   # not defined in this list, uplift will assume its default behaviour 
   # and fail due to the repository being in a dirty state
```

Comment:
What about rewording to indicate why you would use this:
```txt
A list of case sensitive files that will be committed to the repo when the release is created.  Typically these files are managed as part of the release process e.g. generated by `hooks`.  This allows additional information to be included in a release such as a Software Bill Of Materials (SBOM) that is created by an external tool.   
```

Reply:
Thanks - documentation is tough :D